openbas-framework/src/main/java/io/openbas/injector_contract/fields/ContractSelect.java

```diff
@@ -13,17 +14,12 @@
 public class ContractSelect extends ContractCardinalityElement {
 
   private Map<String, String> choices = new HashMap<>();
+  private Map<String, String> choiceInformations = new HashMap<>();
 
   public ContractSelect(String key, String label, ContractCardinality cardinality) {
     super(key, label, cardinality);
   }
 
-  public static ContractSelect selectField(String key, String label, Map<String, String> choices) {
-    ContractSelect contractSelect = new ContractSelect(key, label, ContractCardinality.One);
-    contractSelect.setChoices(choices);
-    return contractSelect;
-  }
-
   public static ContractSelect selectFieldWithDefault(
       String key, String label, Map<String, String> choices, String def) {
```

Comment:
**proposal**
What do you think of having a new class here ?
Choice {
key, value, information
}

To have just one method.